### DISCUSSION TITLE
Motivation for this fork instead of contributing to neo-ltex?

### DISCUSSION THREAD

**Discussion by mxwell-dev** [ltex-plus/vscode-ltex-plus#23] [Q&A] [answered]
Just wondering why ltex+ was founded when neo-ltex already seems to already continue development of the stale original extension?

**Comment by spitzerd** [comment, marked as answer]
Thank you for your valid question. The last meaningful commit on neo-ltex was on 6th February. This was almost 7 months ago. I don´t expect that you have to commit or answer to issues on a daily basis, but severe issues like https://github.com/neo-ltex/vscode-ltex/issues/88 should be addressed in a few weeks. Even a comment on the issue would be fine, but ignoring the issue is not acceptable in my opinion.

Sure, having three LTeX extensions in the VS Code Marketplace is confusing, but I don´t see any alternative.
